cypress/objects/loginPage.js

```diff
@@ -0,0 +1,27 @@
+class LoginPage {
+    visit() {
+        cy.visit('/');
+    }
+
+    getUserFieldSelectorById() {
+        return cy.get('[id="loginForm.userId"]');
+    }
+
+    getRoleFieldSelectorById(){
```

Comment:
Method name could be clearer, because now this method does not return a selector, it returns element which you can click later. I would name this `getRoleField`

Reply:
fixed with commit 'f073c5'